File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/ViewFlashParams.java
package io.appium.espressoserver.lib.model;

import javax.annotation.Nullable;

@SuppressWarnings("unused")
public class ViewFlashParams extends AppiumParams {

    private Integer durationMillis;

    private Integer repeatCount;

    @Nullable
    public Integer getDurationMillis() {
        return durationMillis;
    }

    @Nullable
    public Integer getRepeatCount() {
        return repeatCount;
    }
}

File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/ViewFlashParams.kt
package io.appium.espressoserver.lib.model

val DURATION_MILLIS = 30
val REPEAT_COUNT = 15

data class ViewFlashParams(
    val durationMillis: Int = DURATION_MILLIS,
    val repeatCount: Int = REPEAT_COUNT
) : AppiumParams()
